<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeControlWidget</class>
 <widget class="QWidget" name="TimeControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLCDNumber" name="hour1Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="hour2Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hourLabel">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>h</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="min1Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="min2Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="minLabel">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>m</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="sec1Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="sec2Number">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 56, 127);
color: rgb(206, 243, 255);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
